Cubical/Foundations/Pointed/Base.agda

```diff
@@ -31,6 +31,11 @@ idfun∙ : ∀ {ℓ} (A : Pointed ℓ) → A →∙ A
 idfun∙ A .fst x = x
 idfun∙ A .snd = refl
 
+ua∙ : ∀ {ℓ} {A B : Pointed ℓ} (e : fst A ≃ fst B)
```

Comment:
@ecavallo @mzeuner Didn't we prove this for the SIP at some point? If so, where?